Port messageSearchFilter

Now if you insert value in search results input
![screenshot from 2018-09-25 23-01-49](https://user-images.githubusercontent.com/5389745/46043019-25f4aa00-c117-11e8-840b-8ee245926e01.png)
you get err
```
bundle.js:201133 Error: [$injector:unpr] Unknown provider: messageSearchFilterProvider <- messageSearchFilter
```

--- a/public/app/filters/index.js
+++ b/public/app/filters/index.js
@@ -1,6 +1,8 @@
 import angular from 'angular';
-import PropsFilter from './props-filter';
+import propsFilter from './props-filter';
+import messageSearch from './message-search';
 
 export default angular.module('hepicApp.filters', [
-  PropsFilter.name,
+  propsFilter.name,
+  messageSearch.name
 ]);

--- a/public/app/filters/message-search/index.js
+++ b/public/app/filters/message-search/index.js
@@ -1,0 +1,4 @@
+import angular from 'angular';
+import messageSearch from './message-search';
+
+export default angular.module('hepicApp.filters.messageSearch', []).filter('messageSearch', messageSearch);

--- a/public/app/filters/message-search/message-search.js
+++ b/public/app/filters/message-search/message-search.js
@@ -1,0 +1,28 @@
+const messageSearch = function() {
+  return function(data, grid, query) {
+    var matches = [];
+  
+    //no filter defined so bail
+    if (query === undefined || query === '') {
+      return data;
+    }
+  
+    for (var i = 0; i < data.length; i++) {
+      for (var j = 0; j < grid.columnDefs.length; j++) {
+        var dataItem = data[i];
+        var fieldName = grid.columnDefs[j]['field'];
+  
+        if (dataItem[fieldName] === undefined) continue;
+  
+        if (dataItem[fieldName].toString().search(query) > -1) {
+            var n = dataItem[fieldName].toString().search(query);
+            matches.push(dataItem);
+            break;
+        }
+      }
+    }
+    return matches;
+  }
+};
+
+export default messageSearch;

